src/main/java/com/checkmarx/flow/exception/PullRequestCommentUnknown.java

```diff
@@ -0,0 +1,7 @@
+package com.checkmarx.flow.exception;
+
+public class PullRequestCommentUnknown extends  RuntimeException {
```

Comment:
I think it will be more clearer to declare it's an exception -> PullRequestCommentUnknownException

Reply:
Agree !